app/src/main/java/net/aiscope/gdd_app/ui/capture/CaptureImagePresenter.kt

```diff
@@ -21,6 +27,8 @@ class CaptureImagePresenter(
 
                 view.goToMask(sample.disease, file.absolutePath, sample.nextMaskName())
             }
+            //What if shit falls over tho??
```

Comment:
Language, mister! xD

Reply:
Hahaha temp remark bro. Promise it won't make it into a release 🤞

Reply:
I know, I know, I was just messing with you :P